scripts/post-upgrade.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+set -ex
+
+if [ "$#" -ne 6 ]; then
+	echo "Error : Invalid number of arguments"
+	Usage: "./post-upgrade.sh <pravegacluster name> <pravega-release-name> <bookkeeper-release-name> <version> <namespace> <zk-svc-name>"
+	exit 1
+fi
+
+name=$1
+pname=$2
+bkname=$3
+version=$4
+namespace=$5
+zksvc=$6
+
```

Comment:
Some use of `echo` could be useful to inform the user of what steps are happening / etc

Reply:
done